packages/config/config/devices/0x010f/fgk101.json

```diff
@@ -415,158 +369,277 @@
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32400,
-			"defaultValue": 300
+			"defaultValue": 300,
+			"unit": "seconds"
 		},
 		{
 			"#": "51",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Temperature reports threshold",
-			"description": "This parameter defines the change of temperature in ...",
+			"label": "Temperature Reports Threshold",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 300,
-			"defaultValue": 10
+			"defaultValue": 10,
+			"unit": "0.1 °C"
 		},
 		{
 			"#": "52",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Interval of temperature reports",
-			"description": "This parameter determines how often the temperature ...",
+			"label": "Temperature Reports Interval",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32400,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"unit": "seconds"
 		},
 		{
 			"#": "53",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Temperature offset",
-			"description": "The value to be added to the actual temperature, measured ...",
+			"label": "Temperature Offset",
 			"valueSize": 4,
 			"minValue": -1000,
 			"maxValue": 1000,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"unit": "0.1 °C"
 		},
 		{
-			"#": "54",
+			"#": "54[0x01]",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Temperature alarm reports",
-			"description": "Temperature alarms reported to the Z-Wave controller.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 3,
-			"defaultValue": 0
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "High Temperature Alarm",
+			"valueSize": 1
+		},
+		{
+			"#": "54[0x02]",
+			"$if": "firmwareVersion >= 3.2",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Low Temperature Alarm",
+			"valueSize": 1
 		},
 		{
 			"#": "55",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "High temperature alarm threshold",
-			"description": "If temperature is higher than set value, overheat ...",
+			"label": "High Temperature Alarm Threshold",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 1000,
-			"defaultValue": 540
+			"defaultValue": 540,
+			"unit": "0.1 °C"
 		},
 		{
 			"#": "56",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Low temperature alarm threshold",
-			"description": "If temperature is lower than the set value, underheat ...",
+			"label": "Low Temperature Alarm Threshold",
 			"valueSize": 2,
 			"minValue": -300,
 			"maxValue": 700,
-			"defaultValue": 40
+			"defaultValue": 40,
+			"unit": "0.1 °C"
 		},
 		{
-			"#": "70",
+			"#": "70[0x0001]",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "Scene activation functionality",
-			"description": "The device can trigger scenes using scene IDs assigned ...",
-			"valueSize": 2,
-			"minValue": 0,
-			"maxValue": 2048,
-			"defaultValue": 0
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Scene: Opening Door",
```

Comment:
I think these should all mention the scene ID for the events and the limitation outlined here:
![grafik](https://user-images.githubusercontent.com/17641229/151054263-e74da7e0-45cc-4b2c-8fde-c5c46f800ab2.png)